src/js/common/schemaform/validation.js

```diff
@@ -296,7 +296,7 @@ export function validateAddress(errors, address, formData, schema) {
 
   // Adds error message for state if it is blank and one of the following countries:
   // USA, Canada, or Mexico
-  if (_.includes(address.country)(['USA', 'CAN', 'MEX'])
+  if (['USA', 'CAN', 'MEX'].includes(address.country)
```

Comment:
Could probably make this a Set, too.

Feels weird to say that in a real PR and not a homework eval :-)

Reply:
Hehe, I almost did. 😁 

Will do!